src/lwc/geBatchGiftEntryTable/geBatchGiftEntryTable.html

```diff
@@ -27,58 +27,56 @@ <h1 class="slds-text-title_bold">{title}</h1>
+                                             expected-value={expectedCount}>
                         </c-util-progress-bar>
                     </lightning-layout-item>
-                    <lightning-layout-item size=12 medium-device-size=6
+                    <lightning-layout-item size=12
+                                           medium-device-size=6
                                            class="slds-p-top_small"
                                            padding="horizontal-small">
-                        <c-util-progress-bar
-                                title={geBatchGiftsTotalLabel}
-                                actual-value={total}
-                                expected-value={expectedTotal}
-                                format-style="currency"
-                                currency-code={batchCurrencyISOCode}>
+                        <c-util-progress-bar title={geBatchGiftsTotalLabel}
+                                             actual-value={total}
+                                             expected-value={expectedTotal}
+                                             format-style="currency"
+                                             currency-code={batchCurrencyISOCode}>
                         </c-util-progress-bar>
                     </lightning-layout-item>
                 </lightning-layout>
 
                 <div style="height: 20rem;">
-                    <lightning-datatable
-                            key-field="Id"
-                            data={data}
-                            errors={tableRowErrors}
-                            columns={columns}
-                            hide-checkbox-column="true"
-                            show-row-number-column="true"
-                            onrowaction={handleRowActions}
-                            enable-infinite-loading
-                            onloadmore={loadMoreData}
-                            data-qa-locator="datatable Batch Gifts">
+                    <lightning-datatable key-field="Id"
+                                         data={data}
+                                         errors={tableRowErrors}
+                                         columns={columns}
+                                         hide-checkbox-column="true"
+                                         show-row-number-column="true"
+                                         onrowaction={handleRowActions}
+                                         enable-infinite-loading
+                                         onloadmore={loadMoreData}
+                                         data-qa-locator="datatable Batch Gifts">
                     </lightning-datatable>
                 </div>
-            </div>
-            <div if:false={hasData}>
+            </template>
+            <template if:false={hasData}>
                 <c-util-illustration title="Looks like you haven't entered any gifts yet!"
```

Comment:
I know this isn't part of your WI? Do you know why we're not using a custom label here?